exercises/calculator.cpp

```diff
@@ -10,3 +10,40 @@
   Multiplication: 8
   Division: 2
 */
+
+#include <iostream>
+
+using namespace std;
+
+double sum(double x, double y) {
+  return x + y;
+}
+
+double sub(double x, double y) {
+  return x - y;
+}
+
+double mul(double x, double y) {
+  return x * y;
+}
+
+double div(double x, double y) {
+  return x / y;
+}
+
+int main() {
+  double x, y;
+
+  cout << "Insert the first number: ";
+  cin >> x;
+
+  cout << "Insert the second number: ";
+  cin >> y;
+
+  cout << "SUM: " << sum(x, y) << endl;
+  cout << "Difference: " << sub(x, y) << endl;
+  cout << "Multiplication: " << mul(x, y) << endl;
+  cout << "Division: " << div(x, y) << endl;
+
+  return 0;
```

Comment:
Potresti rendere più esplicito il valore di ritorno del **main** aggiungendo in cima al codice, subito dopo gli "include"

```c++
#define EXIT_SUCCESS 0
```

e alla fine del **main**, al posto di "return 0;"

```c++
return EXIT_SUCCESS;
```

Reply:
Sistemato